src/text.tsx

```diff
@@ -248,17 +248,74 @@ const getTextSizes = ({
     }
 };
 
+export const textValues = {
```

Comment:
I've modularized this because (as you can see in the PR changes), there are several components that were trying to imitate Text2/Text4 values by copy-pasting them. By exporting this object, it can make changes easier in the future

Reply:
I'm curious, can't most of those cases just use `Text2` or `Text4` instead of `Text` ?

Reply:
maybe `textStyles` is a better name

Reply:
updated name to `textProps`